templates/k3s/overlay/opt/corral/k3s/init-server.sh

```diff
@@ -1,12 +1,23 @@
 #!/bin/bash
-set -ex
 
-mkdir -p /etc/rancher/k3s
-cat > /etc/rancher/k3s/config.yaml <<- EOF
-server: https://${CORRAL_api_host}:6443
+config="write-kubeconfig-mode: 644
+server: https://${CORRAL_kube_api_host}:6443
 token: ${CORRAL_node_token}
 tls-san:
   - ${CORRAL_api_host}
+"
+
+mkdir -p /etc/rancher/k3s
+cat >/etc/rancher/k3s/config.yaml <<-EOF
+${config}
 EOF
 
-curl -sfL https://get.k3s.io | KUBERNETES_VERSION=${CORRAL_kubernetes_version} sh -
+curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=${CORRAL_kubernetes_version} sh -
+
+RET=1
+until [ ${RET} -eq 0 ]; do
+  curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=${CORRAL_kubernetes_version} sh -
+  RET=$?
+  journalctl -u k3s
```

Comment:
k3s supports both systemd and OpenRC. I'm ok with this if this is will only run on ubuntu, suse and rhel based distros.

Reply:
So I added the journal for logging purposes, cause initial tries won't be successful until kube-proxy is initialized properly. If you think it's gonna limit the template, I can remove this line

Reply:
I think we're good with the implementation. I've not seen the team using Alpine anyways.